paparazzi/paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -208,8 +208,8 @@ class Paparazzi @JvmOverloads constructor(
   }
 
   @JvmOverloads
-  fun snapshot(view: View, name: String? = null) {
-    takeSnapshots(view, name, 0, -1, 1)
+  fun snapshot(view: View, name: String? = null, timeMillis: Long = 0L) {
```

Comment:
ubernit: should the arg name be more telling?  maybe...

```suggestion
  fun snapshot(view: View, name: String? = null, offsetMillis: Long = 0L) {
```